sql/parser/parse_test.go

```diff
@@ -30,6 +30,10 @@ func TestParse(t *testing.T) {
 	testData := []struct {
 		sql string
 	}{
+		// TODO(tschottdorf): shouldn't these below error?
+		{``},
```

Comment:
The postgres grammar allows empty statements so that multiple semicolons in statement lists are ignored. For example, `SELECT a FROM b; ; SELECT c FROM d;`.  Would be easy to call out an empty statement list as an error in `Parse()`.